app/templates/about.hbs

```diff
@@ -0,0 +1,8 @@
+<html>
+<body class="a-bg">
```

Comment:
The html and body tags should only ever be used once in an html document. Ember already does this in the public/index.html file. Probably want to remove these.